op-challenger/scripts/list_claims.sh

```diff
@@ -11,10 +11,9 @@ echo "Claim count: ${COUNT}"
 for i in $(seq 0 "${COUNT}")
 do
   CLAIM=$(cast call --rpc-url "${RPC}" "${GAME_ADDR}" 'claimData(uint256) returns(uint32 parentIndex, bool countered, bytes32 claim, uint128 position, uint128 clock)' "${i}")
-  SAVEIFS=$IFS   # Save current IFS (Internal Field Separator)
-  IFS=$'\n'      # Change IFS to newline char
-  CLAIM=($CLAIM) # split the string into an array by the same name
-  IFS=$SAVEIFS   # Restore original IFS
+  # Use read -ra to safely split the string into an array named CLAIM, assuming
+  # data is newline-separated.
+  IFS=$'\n' read -ra CLAIM <<< "$CLAIM"
```

Comment:
The `a` option to `read` is a gnu extension that isn't available everywhere (eg on MacOS).  I think we need to go back to the old way of doing this.

Reply:
Ah good catch, reverted to the old approach in 591d885f7